api/v1/storagecluster_types.go

```diff
@@ -105,6 +105,11 @@ type StorageClusterSpec struct {
 	DefaultStorageProfile string `json:"defaultStorageProfile,omitempty"`
 
 	StorageProfiles []StorageProfile `json:"storageProfiles,omitempty"`
+
+	// +kubebuilder:validation:MaxLength=230
+	// StorageClassName is the string to be used as the prefix for
+	// the default storage classes created by ocs-operator.
+	StorageClassName string `json:"storageClassName,omitempty"`
```

Comment:
This should be StorageClassNamePrefix if you are considering it as a prefix.

Reply:
Ack, will update the name